examples/gitlab/postgres.yml

```diff
@@ -0,0 +1,22 @@
+name: postgresql
+containers:
+  - image: sameersbn/postgresql:9.5-3
+    volumeMounts:
+    - name: db
+      mountPath: /var/lib/postgresql
+    env:
+    - name: DB_USER
```

Comment:
variables like  DB_USER,  DB_PASS and  DB_NAME are shared with gitlab container. 
It might be better to use ConfigMap for those.